guides/v2.1/ui_comp_guide/bk-ui_comps.md

```diff
@@ -103,6 +103,18 @@ A particular instance of a UI component is defined primarily by the following:
 3. [Backend/PHP modifiers]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_modifier_concept.html).
 4. Configuration inside the JavaScript classes.
 
+## UI component used in the frontend design area
+
+* Configured through layout XML
+
+* The jsLayout argument is used to specify information
```

Comment:
The provided information is very useful.
However, as a reader, I have doubts about what exact information can we specify using `jsLayout`. 
If you could add an XML example for every case - it would be awesome.
Thank you!

Reply:
I have added the small example of xml layout file for Ui component of frontend area

Reply:
Awesome. Please, adjust the other versions of the documentation as well. As far as I see, you've added the mentioned example only for 2.3.

Thank you!

Reply:
I have added an example  in 2.2 and 2.3 version as well